examples/docker-compose.yml

```diff
@@ -2,13 +2,20 @@ version: '3.4'
 
 services:
   bds:
-    image: itzg/minecraft-bedrock-server
+    image: foleymic/minecraft-bedrock-server
     environment:
       EULA: "TRUE"
       GAMEMODE: survival
       DIFFICULTY: normal
+      SERVER_PORT: 19140
```

Comment:
Was there a reason to change the SERVER_PORT...won't the mc-monitor status check against `--port 19132` fail with this?

Reply:
No, another thing I was testing and forgot to revert.  Will fix.